modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -128,6 +130,20 @@ The following components are affected:
 </ExternalHyperlink>
 ```
 
+## Pill
+
+**PRs:** [#3104](https://github.com/Workday/canvas-kit/pull/3104)
+
+A few changes have been made to `Pill` to ensure proper accessibility and styles.
+
+- The border color on hover has been updated from `licorice400` to `licorice500` to match our design specs.
+
+##### Breaking Changes
+
+- `maxWidth` has been removed from the `usePillModel`. This config was used to style sub components. With the refactor to use `data-part` and stencils, it is no longer needed on the model. You can still apply `maxWidth` on the parent `<Pill>` element and the child elements will be styled accordingly.
```

Comment:
Thoughts on linking to stencils in docs?

Reply:
i like